specification/search/data-plane/Microsoft.Azure.Search.Service/track1/preview/2019-05-06-preview/searchservice.json

```diff
@@ -5234,48 +5689,174 @@
       ],
       "x-ms-enum": {
         "name": "SplitSkillLanguage",
-        "modelAsString": false
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "da",
+            "description": "Danish"
+          },
+          {
+            "value": "de",
+            "description": "German"
+          },
+          {
+            "value": "en",
+            "description": "English"
+          },
+          {
+            "value": "es",
+            "description": "Spanish"
+          },
+          {
+            "value": "fi",
+            "description": "Finnish"
+          },
+          {
+            "value": "fr",
+            "description": "French"
+          },
+          {
+            "value": "it",
+            "description": "Italian"
+          },
+          {
+            "value": "ko",
+            "description": "Korean"
+          },
+          {
+            "value": "pt",
+            "description": "Portuguese"
+          }
+        ]
       },
       "description": "The language codes supported for input text by SplitSkill."
     },
     "EntityRecognitionSkillLanguage": {
-      "type": "string",
-      "enum": [
-        "de",
-        "en",
-        "es",
-        "fr",
-        "it"
-      ],
-      "x-ms-enum": {
-        "name": "EntityRecognitionSkillLanguage",
-        "modelAsString": false
-      },
-      "description": "The language codes supported for input text by EntityRecognitionSkill."
-    },
-    "NamedEntityRecognitionSkillLanguage": {
       "type": "string",
       "enum": [
         "ar",
         "cs",
+        "zh-Hans",
+        "zh-Hant",
         "da",
-        "de",
+        "nl",
         "en",
-        "es",
         "fi",
         "fr",
-        "he",
```

Comment:
As far as I can tell, this one is the only one that's being removed and not moved around. Are we removing support for languages?

Reply:
Yes, unfortunately NER supported it, but entity recognition (NERV2) doesn't.
https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/language-support